apps/teams-test-app/src/components/privateApis/MeetingRoomAPIs.tsx

```diff
@@ -1,104 +1,80 @@
-    };
-    meetingRoom.registerMeetingRoomStatesUpdateHandler(handler);
-  };
+const RegisterMeetingRoomCapabilitiesUpdateHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'registerMeetingRoomCapabilitiesUpdateHandler',
+    title: 'Register MeetingRoom Capabilities Update Handler',
+    onClick: async setResult => {
+      const handler = (meetingRoomCapability: meetingRoom.MeetingRoomCapability): void => {
+        setResult(`Capabilities of meeting room update ${JSON.stringify(meetingRoomCapability)}`);
+      };
+      meetingRoom.registerMeetingRoomCapabilitiesUpdateHandler(handler);
 
-  const checkMeetingRoomCapability = (): void => {
-    if (meetingRoom.isSupported()) {
-      setCapabilityCheckRes('MeetingRoom is supported');
-    } else {
-      setCapabilityCheckRes('MeetingRoom is not supported');
-    }
-  };
+      return generateRegistrationMsg('it is invoked when the live stream state changes');
+    },
+  });
 
-  return (
-    <>
-      <h1>meetingRoom</h1>
-      <BoxAndButton
-        handleClick={getPairedMeetingRoomInfo}
-        output={getPairedMeetingRoomInfoRes}
-        hasInput={false}
-        title="Get Paired MeetingRoom Info"
-        name="getPairedMeetingRoomInfo"
-      />
-      <BoxAndButton
-        handleClickWithInput={sendCommandToPairedMeetingRoom}
-        output={sendCommandToPairedMeetingRoomRes}
-        hasInput={true}
-        title="Send Command To PairedMeeting Room"
-        name="sendCommandToPairedMeetingRoom"
-      />
-      <BoxAndButton
-        handleClick={registerMeetingRoomCapabilitiesUpdateHandler}
-        output={registerMeetingRoomCapabilitiesUpdateHandlerRes}
-        hasInput={false}
-        title="Register MeetingRoom Capabilities Update Handler"
-        name="registerMeetingRoomCapabilitiesUpdateHandler"
-      />
-      <BoxAndButton
-        handleClick={registerMeetingRoomStatesUpdateHandler}
-        output={registerMeetingRoomStatesUpdateHandlerRes}
-        hasInput={false}
-        title="Register MeetingRoom States UpdateHandler"
-        name="registerMeetingRoomStatesUpdateHandler"
-      />
-      <BoxAndButton
-        handleClick={checkMeetingRoomCapability}
-        output={capabilityCheckRes}
-        hasInput={false}
-        title="Check MeetingRoom Capability"
-        name="checkMeetingRoomCapability"
-      />
-    </>
-  );
-};
+const RegisterMeetingRoomStatesUpdateHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'registerMeetingRoomStatesUpdateHandler',
+    title: 'Register MeetingRoom States UpdateHandler',
+    onClick: async setResult => {
+      const handler = (meetingRoomCapability: meetingRoom.MeetingRoomCapability): void => {
+        setResult(`Capabilities of meeting room update ${JSON.stringify(meetingRoomCapability)}`);
+      };
+      meetingRoom.registerMeetingRoomCapabilitiesUpdateHandler(handler);
+
+      return generateRegistrationMsg('it is invoked when the live stream state changes');
```

Comment:
The message here should also be different, something like 'the meeting room states update'.